### DISCUSSION TITLE
No hints in scss files when using tailwind css.

### DISCUSSION THREAD

**Discussion by BolotnikovMS** [zed-industries/zed#16513] [Feature Requests]
Hi everyone, When using tailwind together with scss, no tips appear. Is there any way to fix this?
<img width="485" alt="Снимок экрана 2024-08-20 в 15 22 23" src="https://github.com/user-attachments/assets/80c0a4b0-6477-4a60-91ef-e0c8633f3423">

**Comment by bajrangCoder** [comment]
You should enable that from settings for scss file

**Reply by BolotnikovMS** [reply]
Can you tell me how to do this?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
